db/migrate/20220511212358_clear_revisions_may_eleventh.rb

```diff
@@ -1,9 +0,0 @@
-class ClearRevisionsMayEleventh < ActiveRecord::Migration[7.0]
-  def up
-    Revision.find_by(name: 'datastreams')&.delete
-  end
-
-  def down
-    # nop
-  end
-end
```

Comment:
First I wanted to say that we should keep the migration, just drop its body so we have the table revisions in sync, but actually as you didn't run `db:migrate` and the schema didn't change, it's fine to just drop this. Probably this never even ran on stage.